x-pack/plugins/maps/public/actions/store_actions.js

```diff
@@ -32,6 +33,7 @@ export const MAP_DESTROYED = 'MAP_DESTROYED';
 export const LAYER_DATA_LOAD_STARTED = 'LAYER_DATA_LOAD_STARTED';
 export const LAYER_DATA_LOAD_ENDED = 'LAYER_DATA_LOAD_ENDED';
 export const LAYER_DATA_LOAD_ERROR = 'LAYER_DATA_LOAD_ERROR';
+export const UPDATE_SOURCE_DATA_REQUEST = 'LAYER_DATA_UPDATE_DATA';
```

Comment:
We haven't broken with the convention of naming the constants the same as their value yet. Was this intentional?